layers-bsp.csv

```diff
@@ -1,15 +1,18 @@
 Relative GIT Repository,Layer subdirectory in repository,Tag/Branch to checkout in develop MSC-LDK branch
-thirdparty/yocto,meta,hardknott-5.10.35-2.0.0-msc
-thirdparty/meta-freescale,,hardknott-5.10.35-2.0.0
-thirdparty/meta-freescale-distro,,hardknott-5.10.35-2.0.0
-thirdparty/meta-imx,meta-bsp,hardknott-5.10.35-2.0.0-msc
-thirdparty/meta-imx,meta-sdk,hardknott-5.10.35-2.0.0-msc
-thirdparty/meta-qt5,,hardknott-imx-5.10.35-2.0.0
-thirdparty/meta-nxp-demo-experience,,hardknott-5.10.35-2.0.0
-msc_ol99/meta-msc-arm-extensions,,
-msc_ol99/meta-msc-ldk-marvell,,
-thirdparty/meta-openembedded,meta-oe,hardknott-imx-5.10.35-2.0.0-msc
-thirdparty/meta-openembedded,meta-networking,hardknott-imx-5.10.35-2.0.0-msc
-thirdparty/meta-openembedded,meta-python,hardknott-imx-5.10.35-2.0.0-msc
-thirdparty/meta-openembedded,meta-multimedia,hardknott-imx-5.10.35-2.0.0-msc
-thirdparty/meta-openembedded,meta-gnome,hardknott-imx-5.10.35-2.0.0-msc
+meta-processor-sdk,,dunfell-08.04.01.04-msc
+meta-arago,meta-arago-distro,dunfell-08.04.01.04-msc
+meta-arago,meta-arago-extras,dunfell-08.04.01.04-msc
+meta-qt5,,dunfell-08.04.01.04-msc
+meta-virtualization,,dunfell-08.04.01.04-msc
+meta-ti,,dunfell-08.04.01.04-msc
+meta-openembedded,meta-networking,dunfell-08.04.01.04-msc
```

Comment:
Do you really need these extra branches in meta-openembedded? If so, we would like a name where we can see where it is needed, e.g. imx-<version>, this is from a freescale release.

Reply:
No, Markus, we don't require any special branches of common repos. Any latest stable commit would be Ok.

However, some of the repos on Avnet public repo are not upto date (especially dunfell branch) and last updated in year 2020 like "meta-ti".
Due to that, I have used all the repos from my local github - https://github.com/anilpatelavnet?tab=repositories

Once we update all the common repos to latest one (or please give me write access so that I can push all new patches), we don't require any custom branch here.

Reply:
With MSC-LDK we stick with the initial kirkstone, dunfell, whatever release (4.0.0) and so far don´t follow their minor versions. As we release MSC-LDK for a lot of CPU types we´d run in a nightmare otherwise. For example one wants 4.0.2, another one 4.01, a third one 4.0.0. That is impossible to handle on versioned .bbappend files.